app/forms/steps/case/appeal_details_form.rb

```diff
@@ -0,0 +1,34 @@
+module Steps
+  module Case
+    class AppealDetailsForm < Steps::BaseFormObject
+      include Steps::HasOneAssociation
+      has_one_association :case
+
+      attribute :appeal_lodged_date, :multiparam_date
+      attribute :appeal_with_changes_details, :string
+      attribute :appeal_maat_id, :string
```

Comment:
I'm not performing format validation on MAAT ID, but I believe they confirmed is a number, not alpha-numeric, so we can probably validate format (if entered, as it is optional).
Can do in follow-up PR.